samples/bluetooth/central/src/main.c

```diff
@@ -53,6 +55,10 @@ static void device_found(const bt_addr_le_t *addr, s8_t rssi, u8_t type,
 				BT_LE_CONN_PARAM_DEFAULT, &default_conn);
 	if (err) {
 		printk("Create conn to %s failed (%u)\n", addr_str, err);
+
+		default_conn = NULL;
```

Comment:
What's going on here? If `bt_conn_le_create()` returns failure it should leave the conn pointer untouched. If that's not happening we shouldn't be working around it on the application side like this but fix the actual issue.

Reply:
conn is only assigned when returning success, so this line is not needed.